frontend/certificates/README.md

```diff
@@ -0,0 +1,11 @@
+# Certificates
+
+Copy your certificates to this folder. Two files are expected, both with *.pem* extension.
+
+In order to use certificates to establish secure HTTPS generate certificate and private key files in .pem format. Default names are **certificate.pem** and **privatekey.pem**.
+
+If you want to use HTTPS and you have copied certificates you need to specify 2 build arguments for frontend Dockerfile:
```

Comment:
@galethil  These are ENV, not ARG, so perhaps specify environment variables?

Reply:
@SamuelPull I changed description to "environment variables"